nvflare/app_opt/he/model_decryptor.py

```diff
@@ -119,11 +100,19 @@ def process_dxo(self, dxo: DXO, shareable: Shareable, fl_ctx: FLContext) -> Unio
             fl_ctx: FLContext
 
         Returns: DXO object with decrypted weights
+
         """
+        if self.tenseal_context is None:
+            self.tenseal_context = load_tenseal_context_from_workspace(self.tenseal_context_file, fl_ctx)
```

Comment:
just a note, we might be able to get rid of these lines later.

Reply:
Right. Added a TODO note.